### DISCUSSION TITLE
How to output the ori by Euler-Bunge?

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/FEPX#56] [Q&A] [answered]
How to output the ori by Euler-Bunge instead of Rodrigues

**Comment by mkasemer** [comment, marked as answer]
@siwux The orientations are written in output in the same parameterization and convention as given in the input (specifically the mesh):

```
$ElsetOrientations
nelset rodrigues:active
...
$EndElsetOrientations
```

and potentially the same in the `$ElementOrientations` section, if you supply per-element orientations. Please consult the FEPX documentation for a full description.

You can either edit this manually, or consult the Neper documentation to have your mesh write orientations in the desired parameterization.
